app/controllers/application_controller.rb

```diff
@@ -6,12 +6,12 @@ def hello
 
 
     private
-        # Confirms a logged-in user.
-        def logged_in_user 
-            unless logged_in?
-                store_location
-                flash[:danger] = "Please log in." 
-                redirect_to login_url
-            end 
-        end
+    # Confirms a logged-in user.
+    def logged_in_user 
+        unless logged_in?
```

Comment:
This is one more example where you could reduce lines.
Yes I know, I am being paranoiac.

```
return if logged_in?
store_location
flash[:danger] = "Please log in." 
redirect_to login_url
```

Reply:
It's okay, I didn't except "return if" to be a better alternative for "unless"